sw/device/lib/arch/meson.build

```diff
@@ -21,6 +28,7 @@ sw_lib_arch_fpga_nexysvideo = declare_dependency(
 # Effectively, this is all devices that are not specialized DV
 # testbenches.
 sw_lib_arch_core_devices = {
+  'sim_dv': sw_lib_arch_sim_dv,
```

Comment:
I'm not sure that this is actually a "core device". Is *every* OT binary supposed to be buildable as a DV binary?

Reply:
I'd guess so. It is valuable to be able to run anything that was developed for something else (fpga for example) in dv due to complete observability into the design (mostly for debug). We probably won't always do that, but at least the flow should enable it.

Reply:
Sure. If this is intended to be a "generic" testbench, then I feel fine with it.